### DISCUSSION TITLE
UV-5R Plus

### DISCUSSION THREAD

**Discussion by gartnl** [egzumer/uv-k5-firmware-custom#135] [Q&A] [answered]
Did anyone try this firmware on the UV-5R Plus? I like that model better, and it is basically the same hardware so ..

**Comment by edreams34** [comment]
Yes. It works perfectly with my Quansheng UV-5R+ ! No difference with my UV-K5.

By the way, I would like to thank Egzumer for his excellent work and his selfless sharing with the community! well done man!

**Comment by ErikS-web** [comment, marked as answer]
Yes. See also [this item #109](https://github.com/egzumer/uv-k5-firmware-custom/discussions/109)

**Reply by gartnl** [reply]
Tnx. Now I only need to find a way to build fw with only AM-fix.
